website/content/specification/syntax/constraints.md

```diff
@@ -179,6 +179,16 @@ A constraint may have an OPTIONAL [`@level`](#level) attribute and/or an OPTIONA
 
 If defined, the `<message>` value MUST be a [Metaschema string value](/specification/datatypes#string). It MAY contain a Metapath expression templates that starts with `{`, contains a Metapath expression, and ends with `}`.  When evaluating a template Metapath expression, the context of the Metapath [evaluation focus](#constraint-processing) MUST be the failing value node.
 
+##`matches`
+
+The `matches` constraint is a type of Metaschema constraint that restricts field or flag value(s) based on a component matching the target Metapath expression. Each one of these are discussed below.
+
+A match can be made by 2 different ways based on `@datatypes` and based on a regular expression. 
+
+The `@target` attribute of an `matches` constraint specifies the node in a component definition whose value is equal. The `matches` constraint MUST define a [`target`](#target) with a Metapath expression. The node MUST also match on `@datatype`. 
```

Comment:
You write MUST for `@datatype` but is it an optional attribute or must you use it for all matches constraints? If it is optional and isn't there and you must conform to this additional datatype how does that work?

Secondly what are the possible datatypes? Can you link to the relevant Metaschema data types page?

Reply:
The `@dataytpe` attribute is not mandatory, so I need to remove MUST. However, when the attribute is used, the target must be of type `@datatype`. I have to think of a better way to phrase that.

[Link to Datatypes](https://pages.nist.gov/metaschema/specification/datatypes/)

Reply:
These were updated in pairing I am happy with resolving this.